test/unit/utils_test.rb

```diff
@@ -102,6 +102,12 @@ def teardown
 
     assert_not_equal klass.get_options({skip_defaults: false}, SpreadsheetArchitect), {skip_defaults: true, sheet_name: "Sheet1"}
     assert_not_equal klass.get_options({skip_defaults: false}, Post), {skip_defaults: true, sheet_name: "Posts"}
+
+    ### sets :escape_formulas if not given in a valid way
+    assert_equal klass.get_options({}, SpreadsheetArchitect)[:escape_formulas], true
```

Comment:
I would just test `assert_equal klass.get_options({}, SpreadsheetArchitect)[:escape_formulas], true` the other tests arent really necessary